spring-boot/src/main/java/com/sentrydemos/springboot/Application.java

```diff
@@ -41,6 +41,18 @@ public void initSentry() {
 			options.setEnvironment(environment);
 			options.setTracesSampleRate(1.0);
 			options.setRelease(getRelease());
+			
+			options.setTracesSampler(
+				context -> {
+					//context.getTransactionContext().getName() returns String: GET /products
+					if (context.getTransactionContext().getOperation().equals("http.server") &&
+							context.getTransactionContext().getName().startsWith("OPTIONS")) {
```

Comment:
@sdzhong good you tested this in App Engine because it's possible the 'context' object has different properties on it, due to App Engine providing its own version of the SpringBoot test server.